src/main/java/net/smartcosmos/edge/things/service/things/DeleteThingRestService.java

```diff
@@ -19,6 +20,6 @@
      * @param user the user making the request
      * @return the ResponseEntity
      */
-    @Retryable
+    @Retryable(exclude = { HttpClientErrorException.class }, include = { Throwable.class })
```

Comment:
We'll see where it ends. A custom annotation may be an option going forward.

So far, it's just the first step of custom configuration. Interestingly, if `exclude` is set, `include` is required.